code/modules/mob/living/carbon/human/death.dm

```diff
@@ -45,11 +45,16 @@
 /mob/living/carbon/human/proc/makeSkeleton()
 	ADD_TRAIT(src, TRAIT_DISFIGURED, TRAIT_GENERIC)
 	set_species(/datum/species/skeleton)
-	return 1
+	return TRUE
 
 
 /mob/living/carbon/proc/Drain()
 	become_husk(CHANGELING_DRAIN)
 	ADD_TRAIT(src, TRAIT_BADDNA, CHANGELING_DRAIN)
 	blood_volume = 0
-	return 1
+	return TRUE
+
+/mob/living/carbon/proc/makeUncloneAble()

```

Comment:
Sane capitalization please.

Reply:
Was gonna say something about following the file standard, but its mixed. Underscores time.

Reply:
We don't really have a standard but random capital letter in the middle is just too jarring.

Reply:
Uncloneable is one word isnt it, oof

Reply:
I usually just keep to whatever the above procs is done as, to keep consistent with similar procs. This was just a language failure on my part.